main.js

```diff
@@ -0,0 +1,25 @@
+const menu = document.querySelector('.menu');
+const closeMenu = document.querySelector('.close-menu');
+const sidebar = document.querySelector('.sidebar');
+// const righthandOfLogo = document.querySelector('.righthand-of-logo');
```

Comment:
- [ ] Please, remove this piece of old commented-out code to improve the cleanliness of the project. 💪